src/TypeCatalogGen/project.json

```diff
@@ -4,7 +4,8 @@
     "description": "Generates CorePsTypeCatalog.cs given powershell.inc",
 
     "buildOptions": {
-        "emitEntryPoint": true
+        "emitEntryPoint": true,
+        "debugType": "portable"
```

Comment:
this project only gets built for .NET Core, so putting the `debugType` setting here is good for this project.